.github/workflows/release.yaml

```diff
@@ -347,3 +347,19 @@ jobs:
           branch-suffix: "short-commit-hash"
           base: "main"
 
+  # dispatch release event
+  dispatch:
+    name: Dispatch release event
+    needs: [ 'resolve-tag', 'publish-jar', 'publish-image-dockerhub', 'publish-image-ecr']
+    if: ${{ always() }}
+    continue-on-error: true
```

Comment:
Ok why is this? Should it be sent even if release fails? (maybe add a comment above explaining the reason)